src/Microsoft.VisualStudio.Threading.Analyzers/VSTHRD106UseInvokeAsyncForAsyncEventsAnalyzer.cs

```diff
@@ -65,27 +59,24 @@ public override void Initialize(AnalysisContext context)
             context.EnableConcurrentExecution();
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze);
 
-            context.RegisterCodeBlockStartAction<SyntaxKind>(ctxt =>
+            context.RegisterOperationBlockStartAction(context =>
```

Comment:
Interesting. I didn't realize you could overlap an outer scoped variable with a lambda variable.

Reply:
New for C# 8.